test/solvers/advanced/proximal_utils_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for the utility functions for proximal operators."""
+
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# External
+import numpy as np
+import pytest
+
+# Internal
+import odl
+import odl.solvers as odls
+
+from odl.util.testutils import all_almost_equal
+
+# Places for the accepted error when comparing results
+PLACES = 8
+
+
+def test_proximal_translation():
+    """Test for the proximal of a translation: prox[f(. - g)]"""
```

Comment:
Not quite sure I liks this test design, makes the tests circular. What if  `proximal_l2` was implemented using `proximal_translation`? Then it would be true by definition.